exportReport/FrameExportReport.java

```diff
@@ -0,0 +1,163 @@
+	final String					PANEL_RISK			= Constant.messages.getString("exportReport.panel.risk");
+	final String					PANEL_DETAILS		= Constant.messages.getString("exportReport.panel.details");
+	final String					cardItems[]			=
+	{
+			PANEL_SOURCE, PANEL_RISK, PANEL_DETAILS
+	};
+
+	public FrameExportReport(ExtensionExportReport extension, PanelSource cardSource, PanelAlertRisk cardAlertRisk, PanelAlertDetails cardAlertDetails)
+	{
+		this.extension = extension;
+		this.setTitle(Constant.messages.getString("exportReport.menu.title"));
+		this.addComponentToPane(this.getContentPane(), cardSource, cardAlertRisk, cardAlertDetails);
+	}
+
+	public void addComponentToPane(Container contentPane, PanelSource cardSource, PanelAlertRisk cardAlertRisk, PanelAlertDetails cardAlertDetails)
+	{
+
+		SpringLayout sl_contentPane = new SpringLayout();
+		contentPane.setLayout(sl_contentPane);
+
+		pnlCards = new JPanel(new CardLayout());
+		pnlCards.add(cardSource, PANEL_SOURCE);
+		pnlCards.add(cardAlertRisk, PANEL_RISK);
+		pnlCards.add(cardAlertDetails, PANEL_DETAILS);
+
+		pnlCards.setBorder(new CompoundBorder()); // -Give the cards panel a border
+		sl_contentPane.putConstraint(SpringLayout.NORTH, pnlCards, 10, SpringLayout.NORTH, contentPane);
+		sl_contentPane.putConstraint(SpringLayout.WEST, pnlCards, 10, SpringLayout.WEST, contentPane);
+		sl_contentPane.putConstraint(SpringLayout.SOUTH, pnlCards, 345, SpringLayout.NORTH, contentPane);
+		sl_contentPane.putConstraint(SpringLayout.EAST, pnlCards, 370, SpringLayout.WEST, contentPane);
+		contentPane.add(pnlCards);
+
+		pnlNav = new PanelNav(cardItems.length); // -Use the constructor to initialize the
+		// navigation panel.
+		sl_contentPane.putConstraint(SpringLayout.NORTH, pnlNav, 20, SpringLayout.SOUTH, pnlCards);
+		sl_contentPane.putConstraint(SpringLayout.WEST, pnlNav, 10, SpringLayout.WEST, contentPane);
+		sl_contentPane.putConstraint(SpringLayout.SOUTH, pnlNav, 100, SpringLayout.SOUTH, pnlCards);
+		sl_contentPane.putConstraint(SpringLayout.EAST, pnlNav, 0, SpringLayout.EAST, pnlCards);
+
+		// Assign buttons from the navigation pane action listeners.
+		this.btnBack = pnlNav.getBack();
+		this.btnBack.addActionListener(this);
+		this.btnNext = pnlNav.getNext();
+		this.btnNext.addActionListener(this);
+
+		contentPane.add(pnlNav); // -Add the navigation panel to the content
+	}
+
+	private int incCard(int curCard)
+	{
+		if (curCard < cardItems.length - 1)
+			curCard = curCard + 1;
+		return curCard;
+	}
+
+	private int decCard(int curCard)
+	{
+		if (curCard > 0)
+			curCard = curCard - 1;
+		return curCard;
+	}
+
+	@Override
+	public void actionPerformed(ActionEvent e)
+	{
+		if (e.getSource() == btnBack)
+		{
+			curCard = decCard(curCard);
+			CardLayout cl = (CardLayout) (pnlCards.getLayout());
+			cl.show(pnlCards, cardItems[curCard]);
+			if (curCard > 0)
+			{
+				pnlNav.setProgress(curCard + 1);
+				btnNext.setText("Next");
```

Comment:
It should and it actually is but that is an oversight on my end in this file, i forgot to use it. Good catch!